.gitignore

```diff
@@ -10,7 +10,7 @@ target
 .idea
 *.iml
 *.log
-esvm
+/esvm
```

Comment:
I looked briefly, but couldn't find an answer to what pre-pending esvm with / does - is it just to ensure it ignores that path? Would `esvm/**` do the same thing?